core/src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -31,18 +42,31 @@
 
     public static final String NAME = "fuzzy";
 
-    private final String name;
+    /** Default maximum edit distance. Defaults to AUTO. */
+    public static final Fuzziness DEFAULT_FUZZINESS = Fuzziness.AUTO;
+
+    /** Default number of initial characters which will not be “fuzzified”. Defaults to 0. */
+    public final static int DEFAULT_PREFIX_LENGTH = FuzzyQuery.defaultPrefixLength;
+
+    /** Default maximum number of terms that the fuzzy query will expand to. Defaults to 50. */
+    public final static int DEFAULT_MAX_EXPANSIONS = FuzzyQuery.defaultMaxExpansions;
+
+    /** Default as to whether transpositions should be treated as a primitive edit operation, 
+     * instead of classic Levenshtein algorithm. Defaults to false. */
+    public final static boolean DEFAULT_TRANSPOSITIONS = false;
+
+    private final String fieldName;
 
     private final Object value;
 
-    private Fuzziness fuzziness;
+    private Fuzziness fuzziness = DEFAULT_FUZZINESS;
 
-    private Integer prefixLength;
+    private Integer prefixLength = DEFAULT_PREFIX_LENGTH;
 
-    private Integer maxExpansions;
+    private Integer maxExpansions = DEFAULT_MAX_EXPANSIONS;
 
     //LUCENE 4 UPGRADE  we need a testcase for this + documentation
-    private Boolean transpositions;
+    private Boolean transpositions = DEFAULT_TRANSPOSITIONS;
```

Comment:
this would make sense especially given that their setters accept primitive types